lib/opam_cmd.ml

```diff
@@ -23,6 +23,8 @@ let pp_header = Fmt.(styled `Blue string)
 
 let header = "==> "
 
+let pp_plural = Fmt.using (fun xs -> if List.length xs < 2 then "" else "s") Fmt.string
```

Comment:
Better, thanks. I've taken this and switched the order of patterns to avoid needing 3. Feel free to go back to your idea before merging, as you prefer :slightly_smiling_face: